### DISCUSSION TITLE
Is there anyway to make the scrollbar dynamic?

### DISCUSSION THREAD

**Discussion by isayakmondal** [pabli24/DMScrollbar#3] [Ideas]
I really like the look and feel of the scrollbar on the dark mode webpages, but it doesn't quite match with light mode webpages. So I was wondering if you could add a feature that will change the look and feel of the scrollbar according to the theme (light or dark) of the webpage.

**Comment by pabli24** [comment]
https://github.com/pabli24/ultimate-scrollbar
